build/config/rmi_pacta_2023q4_pa2024ch.json

```diff
@@ -9,6 +9,16 @@
       "EN",
       "DE",
       "FR"
+    ],
+    "peer_group": [
+      "insurance",
+      "bank",
+      "pension_fund",
+      "asset_manager",
+      "civil_society",
+      "politician",
+      "researcher",
+      "other"
```

Comment:
These are all valid user groups on the platform, but we can probably drop some of these that are less used.

Do also note that the latest changes have removed `bank` and `insurance`, since the build was failing on trying to create an executive summary. See #271